### PR TITLE
Use the default.html layout on the homepage

### PR DESCRIPTION
**Pull request by keithbloom**
Hi Mike

I had a tinker around on the train and made two changes

- Updated the nav to look like the main bootstrap style
- Change the index.html to use the default template

The change to the index.html is the biggest of the two here. The advantage is that the main page structure is now in the `default.html`. It does mean that I had to add some variables which control the display of the Jumbotron on the home page.

Hope it helps.

### CODE REVIEW

**Comment by mikehadlow** [comment]
Great work Keith. I'm really enjoying your train journeys :)

File: _layouts/default.html [hunk 1]
```diff
@@ -12,30 +10,35 @@
...
+      </div>
     </nav>
 
     <main role="main">
-      <div class="container">
-        <br/>    
-      </div>
 
-      <div class="container">
-          <h1>{{ page.title }}</h1>
```
Comment (mikehadlow) [inline _layouts/default.html:32]: Now there's no page title at the top of each page, it's a little hard to know where one is on the site. I can see why this looks crappy from a design point of view, but could we put it back somewhere?

Comment (keithbloom) [reply _layouts/default.html:32]: Oops, I didn't realise I removed that. I'll put it back

Comment (mikehadlow) [reply _layouts/default.html:32]: No worries :)

**Pull request merged**